trace/client_test.go

```diff
@@ -0,0 +1,47 @@
+package trace
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+)
+
+type server struct {
+	LastSpanID SpanID
+}
+
+func (s server) ServeHTTP(rw http.ResponseWriter, r *http.Request) {
+	span := ParseSpanID(r)
+	fmt.Fprintf(rw, "%#v", span)
+}
+
+func TestClient(t *testing.T) {
+	s := server{}
+	ts := httptest.NewServer(s)
+	defer ts.Close()
+	or, err := http.NewRequest("GET", "example.com", nil)
+	if err != nil {
+		t.Errorf("Failed to create Request")
```

Comment:
Maybe use `t.Fatalf(...)`? There is no point continuing this test if this fails.